### PR TITLE
ignore empty session files

### PR DESCRIPTION
**Pull request by x4base**
!data is true means that the file is just somehow empty. I think there is not much the user can do here. Maybe it's better just to ignore this and let a new session to be started. Or it will just hang all the requests. We even ignored it when err is not empty at line #136, right? It means we really don't care what happens.

### CODE REVIEW

**Comment by lexoyo** [comment]
hmm nice
maybe something like this would be simpler

```
           if (!data) {
              // no session file
              // the file is somehow empty. not much we can do, just continue.
              return fn();
            }
```

**Comment by x4base** [comment]
That's cool. I pushed again.

File: lib/connect-fs.js [hunk 1]
```diff
@@ -166,7 +165,7 @@ module.exports = function(session){
     try {
       var maxAge = sess.cookie.maxAge;
       var now = new Date().getTime();
-      var expired = maxAge ? now + maxAge : now + oneDay;
+      var expired = maxAge ? now + maxAge : now + oneDay * 1000;
```
Comment (lexoyo) [inline lib/connect-fs.js]: @x4base we should set oneDay to 86400000 from start, at line 20, what do you think?

Comment (x4base) [inline lib/connect-fs.js]: sure, that should be easier. Updated

**Comment by lexoyo** [comment]
:+1: nice thx

**Pull request merged**